config/application.rb

```diff
@@ -24,10 +24,14 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
```

Comment:
This needs to be updated. I'm surprised a `new_framework_defaults_6_1.rb` file didn't get generated either? 🤔

Reply:
Thanks, I've updated.  Looks like I didn't commit `new_framework_defaults_6_1.rb`.  Do we need it, since it'll just be removed in another commit?

Reply:
In my [reviews for the other devs](https://github.com/alphagov/maslow/pull/735#pullrequestreview-760596796), I've been advising we keep it as an audit trail, and ideally commenting which options are applicable and which are not, like I've done [here](https://github.com/alphagov/release/pull/858). But no, if none of the new defaults cause anything to fail, then there's no particular need to commit-then-remove the defaults file, so it's not a blocker for me 👍